google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -358,6 +364,18 @@ private Path getFileFromResourcePath(String resourcePath) {
     return appYaml;
   }
 
+  private Optional<Path> getProjectJavaSdk(Project project) {
+    ProjectSdksModel projectSdksModel = new ProjectSdksModel();
```

Comment:
is ProjectRootManager.getInstance(project).getProjectSdk() the better API to use here?